{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"ExceptionsHandler","level":"error","message":"Work order not found","stack":["Error: Work order not found\n    at WorkOrdersController.findOne (C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\src\\work-orders\\work-orders.controller.ts:35:13)\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-execution-context.js:46:28\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-proxy.js:9:17"]}
{"context":"ExceptionsHandler","level":"error","message":"Work order not found","stack":["Error: Work order not found\n    at WorkOrdersController.findOne (C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\src\\work-orders\\work-orders.controller.ts:35:13)\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-execution-context.js:46:28\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-proxy.js:9:17"]}
{"context":"ExceptionsHandler","level":"error","message":"Work order not found","stack":["Error: Work order not found\n    at WorkOrdersController.findOne (C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\src\\work-orders\\work-orders.controller.ts:35:13)\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-execution-context.js:46:28\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-proxy.js:9:17"]}
{"context":"ExceptionsHandler","level":"error","message":"Work order not found","stack":["Error: Work order not found\n    at WorkOrdersController.findOne (C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\src\\work-orders\\work-orders.controller.ts:35:13)\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-execution-context.js:46:28\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-proxy.js:9:17"]}
{"context":"ExceptionsHandler","level":"error","message":"Work order not found","stack":["Error: Work order not found\n    at WorkOrdersController.findOne (C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\src\\work-orders\\work-orders.controller.ts:35:13)\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-execution-context.js:46:28\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-proxy.js:9:17"]}
{"context":"ExceptionsHandler","level":"error","message":"Work order not found","stack":["Error: Work order not found\n    at WorkOrdersController.findOne (C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\src\\work-orders\\work-orders.controller.ts:35:13)\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-execution-context.js:46:28\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-proxy.js:9:17"]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"ExceptionsHandler","level":"error","message":"Work order not found","stack":["Error: Work order not found\n    at WorkOrdersController.findOne (C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\dist\\work-orders\\work-orders.controller.js:31:19)\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-execution-context.js:46:28\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-proxy.js:9:17"]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"ExceptionsHandler","level":"error","message":"Work order not found","stack":["Error: Work order not found\n    at WorkOrdersController.findOne (C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\dist\\work-orders\\work-orders.controller.js:31:19)\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-execution-context.js:46:28\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-proxy.js:9:17"]}
{"context":"ExceptionsHandler","level":"error","message":"Work order not found","stack":["Error: Work order not found\n    at WorkOrdersController.findOne (C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\dist\\work-orders\\work-orders.controller.js:31:19)\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-execution-context.js:46:28\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-proxy.js:9:17"]}
{"context":"ExceptionsHandler","level":"error","message":"Work order not found","stack":["Error: Work order not found\n    at WorkOrdersController.findOne (C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\dist\\work-orders\\work-orders.controller.js:31:19)\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-execution-context.js:46:28\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-proxy.js:9:17"]}
{"context":"ExceptionsHandler","level":"error","message":"Work order not found","stack":["Error: Work order not found\n    at WorkOrdersController.findOne (C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\dist\\work-orders\\work-orders.controller.js:31:19)\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-execution-context.js:46:28\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-proxy.js:9:17"]}
{"context":"ExceptionsHandler","level":"error","message":"Work order not found","stack":["Error: Work order not found\n    at WorkOrdersController.findOne (C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\dist\\work-orders\\work-orders.controller.js:31:19)\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-execution-context.js:46:28\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-proxy.js:9:17"]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"ExceptionsHandler","level":"error","message":"Work order not found","stack":["Error: Work order not found\n    at WorkOrdersController.findOne (C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\dist\\work-orders\\work-orders.controller.js:31:19)\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-execution-context.js:46:28\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-proxy.js:9:17"]}
{"context":"ExceptionsHandler","level":"error","message":"Work order not found","stack":["Error: Work order not found\n    at WorkOrdersController.findOne (C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\dist\\work-orders\\work-orders.controller.js:31:19)\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-execution-context.js:46:28\n    at async C:\\Users\\Cooper\\Documents\\Code\\WorkOrders\\apps\\backend\\node_modules\\@nestjs\\core\\router\\router-proxy.js:9:17"]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
{"context":"NestApplication","level":"error","message":"Error: listen EADDRINUSE: address already in use :::3001","stack":[null]}
